Scripts/release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh 
+
+CONTENTFUL_SDK_VERSION=$(cat .env | grep "CONTENTFUL_SDK_VERSION" | cut -d "=" -f 2)
```

Comment:
❤️

Reply:
thanks again for the help @dlitvakb

Reply:
I would love you to try using `source .env`. I just tried it locally and it did work on my bash.